README.md

```diff
@@ -1 +1,51 @@
-# build-and-push-action
\ No newline at end of file
+# The Build and Push GitHub Action
+
+## Action Inputs
+
+### Registry information
+
+| Input Name           | Description                                                 | Default value |
+|----------------------|-------------------------------------------------------------|---------------|
+| `registry`           | Registry to push container image to.                        | **required**  |
+| `registry_namespace` | Namespace of the registry, where the image would be pushed. | **required**  |
+| `registry_username`  | Login to specified registry.                                | **required**  |
+| `registry_token`     | Access token to specified registry.                         | **required**  |
+
+### Build information
+
+| Input Name       | Description                                                     | Default value |
+|------------------|-----------------------------------------------------------------|---------------|
+| `version `       | Version of the built image.                                     | **required**  |
```

Comment:
I don't follow, what does it mean, version? The name is strictly used by SCLORG images. But let's say, that this action will be used only by our organization.

Reply:
This should mean a version of the included binary, e.g., 3.9 for the s2i-python container, etc.
Should I explain it more clearly in the readme?

Reply:
> This should mean a version of the included binary, e.g., 3.9 for the s2i-python container, etc. Should I explain it more clearly in the readme?

Yes, definitely. It could be understandable. I would say a directory or something better. The problem I see here is that we have a `docker_context` and `version`.

Reply:
Done.

Reply:
Hmm, it is unlucky, that in this action has the version input two meanings, and the one you mentioned (Path to Dockerfile without Dockerfile file) is hidden for the user and could cause some problems in the future. I have added some more explanation to the readme and I will also create an issue for this, as it seems important enough to me.